examples/_extract_wind_speed_at_points.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import floris.tools.visualization as wakeviz
+from floris.tools import FlorisInterface
+from floris.tools.visualization import (
+    calculate_horizontal_plane_with_turbines,
+    visualize_cut_plane,
+)
+
+
+"""
+Test the use of add points
+"""
+
+# Initialize FLORIS with the given input file via FlorisInterface.
+# For basic usage, FlorisInterface provides a simplified and expressive
+# entry point to the simulation routines.
+fi = FlorisInterface("inputs/gch.yaml")
+
+
+# Set up a two-turbine farm
+D = 126
+fi.reinitialize(layout_x=[0, 3*D], layout_y=[0, 3*D])
+
+fig, ax = plt.subplots(1,2)
+fig.set_size_inches(10,4)
+ax[0].scatter(fi.layout_x, fi.layout_y, color="black", label="turbine")
+
+# Set the wind direction to run 360 degrees
+wd_array = np.arange(0, 360, 1)
+fi.reinitialize(wind_directions=wd_array)
+
+# Grab the power before we change anything
+fi.calculate_wake()
```

Comment:
`fi.sample_flow_at_points` runs the wake calculation. Is this line needed?

Reply:
`fi.sample_flow_at_points()` was set up to raise an error if `calculate_wake()` had not been run; however, that was in the previous iteration that assumed that the flow would need to be solved before `full_flow_sequential_solver()` was called. I've now commented out the block that raises the error and can now remove this `calculate_wake()`.